Documentation/Signoffs/NavSystem/Detail Design.md

```diff
@@ -0,0 +1,34 @@
+## Function of the Subsystem
+
+The Navigation System serves as the central control hub, responsible for directing the robot as it traverses its predefined course.
+The navigation system will use an array of infrared avoidance sensors to detect the yellow line. These sensors output low on black and dark colors, and high on white and light colors. The array will output digital bits 
+
+## Constraints
```

Comment:
Put the constraints into a table and number them. Also, this seems like a short list. This is a sensing system so immediately we need constraints for sampling rate and the input (sensed colors).